packages/flutter/lib/src/rendering/sliver_multi_box_adaptor.dart

```diff
@@ -224,23 +250,81 @@ abstract class RenderSliverMultiBoxAdaptor extends RenderSliver
 
   bool _debugAssertChildListLocked() => childManager.debugAssertChildListLocked();
 
+  /// Verify the child list index is in strictly increasing order.
+  ///
+  /// This is a no-op for non-debug builds.
```

Comment:
"no-op" is a bit informal, consider something like "This has no effect in release builds."

Reply:
fixed